README.md

```diff
@@ -572,14 +567,168 @@ Here's how you can configure it:
 
 This configuration option may not be changed after installation.
 
+### Recovery and Health Checks
+
+You can enable automated replacement of brokers and configure the circumstances under which they are replaced.
+
+#### Enable Broker Replacement
+
+To enable automated replacement: <!-- what's all this about data loss, then? -->
+
+* **DC/OS CLI options.json**:
+
+```json
+    {
+        "enable_replacement":{
+            "description":"Enable automated replacement of Brokers. WARNING: May cause data loss. See documentation.",
+            "type":"boolean",
+            "default":false
+        }
+    }
+```
+
+* **DC/OS web interface**: Set the environment variable `ENABLE_REPLACEMENT` = true to enable replacement. <!-- this doesn't follow the convention, but nothing that follows makes sense if this is set to false -->
```

Comment:
I'm suggesting a user set this to "true," as opposed to the default since none of the instructions that follow are relevant if this is false.